storage/storage-interface/src/finality_view.rs

```diff
@@ -82,26 +100,86 @@ mod tests {
     #[test]
     fn test_get_latest_ledger_info() {
         let view = FinalityView::new(MockDbReaderWriter);
+
         let ledger_info = view.get_latest_ledger_info_option().unwrap();
         assert_eq!(ledger_info, None);
-        let fin_ledger_info =
-            LedgerInfoWithSignatures::new(LedgerInfo::dummy(), AggregateSignature::empty());
-        view.set_finalized_ledger_info(fin_ledger_info.clone())
-            .unwrap();
-        let ledger_info = view.get_latest_ledger_info().unwrap();
-        assert_eq!(ledger_info, fin_ledger_info);
+        let blockheight = 1;
+
+        // Set the finalized ledger info
+        view.set_finalized_ledger_info(blockheight).unwrap();
+
+        // Capture the block event once
+        let (_start_ver, end_ver, block_event) =
+            view.get_block_info_by_height(blockheight).unwrap();
```

Comment:
I'd rather use a clone of the mock to get the expected values than go through the object under test, just to prevent any masked problems if we implement more view methods.

Reply:
Will update.